content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/log-in-chainguard-enforce/index.md

```diff
@@ -106,6 +106,42 @@ Valid! Id: <Relevant ID here>
 
 You are now logged in to Chainguard Enforce and can use `chainctl` for your account. 
 
+## Authenticating through another device with `chainctl`
+
+You can authenticate and log into Chainguard Enforce with any device (this will typically be a smartphone) when you cannot access your present working device's browser.
+
+To authenticate, you will browse to the provided URL on an alternate device.
+
+First, authenticate with `chainctl`.
```

Comment:
```suggestion
First, authenticate with `chainctl` on a device that lacks a web browser.
```

I'm not sure if this is too much, thought it might be helpful to clarify

Reply:
Since it includes containers, maybe "device" is not the right word. Maybe: 

```suggestion
First, authenticate with `chainctl`. In this workflow, we are assuming you are not able to access a browser, so you may be authenticating from a virtual machine or container. 
```